bitsail-connectors/connector-kudu/src/main/java/com/bytedance/bitsail/connector/kudu/source/split/KuduSourceSplit.java

```diff
@@ -42,7 +42,29 @@ public class KuduSourceSplit implements SourceSplit {
 
   private final String splitId;
 
-  private List<byte[]> serializedPredicates;
+  public byte[] getSerializedScanToken() {
+    return serializedScanToken;
+  }
+
+  public void setSerializedScanToken(byte[] serializedScanToken) {
+    this.serializedScanToken = serializedScanToken;
+  }
+
+  public String[] getLocations() {
+    return locations;
+  }
+
+  public void setLocations(String[] locations) {
+    this.locations = locations;
+  }
+
+  private List<byte[]> serializedPredicates = new ArrayList<>();
```

Comment:
It is recommended to move these members to the beginning of the class.

Reply:
OK